### DISCUSSION TITLE
Error when executng CarlaUE4.exe.

### DISCUSSION THREAD

**Discussion by hemant792** [carla-simulator/carla#7286] [Q&A]
i installed Carla as per the instructions given on https://carla.readthedocs.io/en/0.9.15/build_windows/. but when i run  "C:\carsim\carla\Unreal\CarlaUE4\Binaries\Win64\CarlaUE4.exe"  its shows error.
why is it happening? Everythings was done as per their webpage instructions. when i ran "make launch" command, it completed successfully but when it was loading unreal engine at the end, it got stuck at 95% for more than half an hour so i had to close it manually. and then when i run carlaUE4.exe i found this error.
![Screenshot 2024-03-23 112449](https://github.com/carla-simulator/carla/assets/153008405/832c783c-e4f2-4986-b537-ee1621f312e8)

**Comment by Blyron** [comment]
Getting stuck at 95% is common, let it work, 
Unreal Engine launch has 3 "%" it appears to be stucked but it is not. 45% when loading assets and 95% when compiling shaders. It takes a while. Please do not use the exe, always use the make launch for a non package.
